src/time_series_container.jl

```diff
@@ -73,72 +71,69 @@ end
 function get_time_series(
     ::Type{T},
     container::TimeSeriesContainer,
-    initial_time::Dates.DateTime,
     label::AbstractString,
 ) where {T <: TimeSeriesMetadata}
-    key = TimeSeriesKey(T, initial_time, label)
+    key = TimeSeriesKey(T, label)
     if !haskey(container.data, key)
         throw(ArgumentError("time_series $key is not stored"))
     end
 
     return container.data[key]
 end
 
-function get_time_series_initial_times(container::TimeSeriesContainer)
-    initial_times = Set{Dates.DateTime}()
-    for key in keys(container.data)
-        push!(initial_times, key.initial_time)
-    end
+function get_time_series_initial_times(ts_metadata::TimeSeriesMetadata)
+    initial_time_stamp = get_initial_time_stamp(ts_metadata)
+    interval = get_interval(ts_metadata)
+    count = get_count(ts_metadata)
+    return collect(range(initial_time_stamp; length = count, step = interval))
+end
 
-    return sort!(Vector{Dates.DateTime}(collect(initial_times)))
+function get_time_series_initial_times(container::TimeSeriesContainer)
+    ts_metadata = first(values(container.data))
+    return get_time_series_initial_times(ts_metadata)
```

Comment:
I'm not sure this method makes sense.  The user could store static time series and forecasts and so the answer may not be true for all.  If the users stores one forecast with labelX and another with labelY, we don't technically need to impose a requirement that those forecasts have the same initial times.  I'm not saying that that isn't a good restriction; just pointing out that each forecast is now completely independent.